vnext/Microsoft.ReactNative/Views/Image/ReactImage.cpp

```diff
@@ -164,6 +165,9 @@ winrt::fire_and_forget ReactImage::SetBackground(bool fireLoadEndEvent) {
   if (auto strong_this{weak_this.get()}) {
     if (strong_this->m_useCompositionBrush) {
       const auto compositionBrush{ReactImageBrush::Create()};
+      strong_this->Loaded([=](auto &&, auto &&) -> auto {
```

Comment:
Would you mind adding a comment, why the ordering of adding this is okay, i.e. that the loaded event fire will happen before the brushes `GetOrCreateSurfaceBrush` is called (or any usage of the compositor in the brush for that matter...)

Reply:
```suggestion
      // GetCompositor relies on the element's XamlRoot which is only set once the object enters the tree.
      // This in turn happens before GetOrCreateSurfaceBrush
      strong_this->Loaded([=](auto &&, auto &&) -> auto {
```